src/dotnet-publish-iis/project.json

```diff
@@ -14,12 +14,21 @@
     "Microsoft.Extensions.Configuration.Json": "1.0.0-*",
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
     "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
-    "NETStandard.Library": "1.5.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
 
   "frameworks": {
-    "dnxcore50": {
+    "netstandard1.5": {
```

Comment:
@brthor does mention in this [issue](https://github.com/dotnet/cli/issues/1970) that tools should target `netstandard1.5`. I guess the suggestion has been changed? changed?